TSC failed since version 2.6.0

**Scope**: Form renderer

**Description**
I am trying to bump the version of the data driven form to the latest version. I got the following error when I am running tsc on my project:

```
node_modules/@data-driven-forms/react-form-renderer/dist/cjs/index.d.ts:48:1 - error TS2308: Module './renderer-context' has already exported a member named 'FormOptions'. Consider explicitly re-exporting to resolve the ambiguity.

48 export * from './wizard-context';
```

I have tried to install version by version. The issue presented from version 2.6.0.

--- a/packages/react-form-renderer/src/files/wizard-context.d.ts
+++ b/packages/react-form-renderer/src/files/wizard-context.d.ts
@@ -1,21 +1,19 @@
-import { ReactNode } from 'react';
-import { FormApi } from 'final-form';
-import Field from './field';
-import { AnyObject } from './common';
-
-export interface FormOptions extends FormApi {
-  registerInputFile?: (name: string) => void;
-  unRegisterInputFile?: (name: string) => void;
-  onCancel?: (values: object, ...args: any[]) => void;
-  onReset?: () => void;
-  handleSubmit: () => Promise<AnyObject | undefined> | undefined;
-  clearedValue?: any;
-  renderForm: (fields: Field[]) => ReactNode[];
-}
+import { FormOptions } from './renderer-context';
 
 export interface WizardContextValue {
   formOptions: FormOptions;
-
+  crossroads: string[];
+  currentStep: string;
+  handlePrev: Function;
+  onKeyDown: Function;
+  jumpToStep: Function;
+  setPrevSteps: Function;
+  handleNext: Function;
+  navSchema: Object;
+  activeStepIndex: number;
+  maxStepIndex: number;
+  isDynamic: boolean;
+  prevSteps: string[];
 }
 
 interface WizardContext {

